src/components/Canvas/index.ts

```diff
@@ -510,6 +528,57 @@ export default class Canvas extends EventDispatcher {
     );
   }
 
+  drawCheckerboard() {
```

Comment:
Also using another canvas for drawing a background might not work well along with the pixel grids since we have lots of pan zoom happening in the pixel grid canvas. We might need to later change the logic to have the pattern drawn directly on the original canvas. For now, let's stick to this version since it is easy to understand🤩